.github/workflows/main.yml

```diff
@@ -0,0 +1,96 @@
+name: Main
+
+on:
+  push:
+    branches: [ main, wip ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Every twelve (12) hours.
+    # @see https://crontab.guru/every-12-hours
+    - cron: 0 */12 * * *
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: "${{ matrix.os }}-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu ]
+        php: [ 8.1, 8.2, 8.3 ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4 # https://github.com/marketplace/actions/checkout
+
+      - name: Set up PHP CLI
+        uses: shivammathur/setup-php@v2 # https://github.com/marketplace/actions/setup-php-action
+        with:
+          php-version: "${{ matrix.php }}"
+
+      - name: Create project
+        run: |
+          set -o xtrace
+
+          # Prepare the target directory.
+          sudo mkdir -p /var/www
+          sudo rm -rf /var/www/*
+          sudo chmod 777 /var/www
+
+          composer create-project \
+            --stability=dev \
+            --repository='{"type":"vcs","url":"'git@github.com:phenaproxima/drupal-suite.git'"}' \
+            --ansi \
+            drupal/starshot-project \
+            /var/www
+
+          # Enable verbose error logging (debug mode).
+          echo "if (file_exists(__DIR__ . '/settings.local.php')) {
+            include __DIR__ . '/settings.local.php';
+          }" | sudo tee -a /var/www/web/sites/default/settings.php
+          sudo cp /var/www/web/sites/example.settings.local.php \
+            /var/www/web/sites/default/settings.local.php
+
+          # Configure database settings.
+          echo "\$databases['default']['default'] = array(
+            'driver' => 'sqlite',
+            'database' => 'db.sqlite',
+            'prefix' => '',
+            'namespace' => 'Drupal\\Core\\Database\\Driver\\sqlite',
+          );" | sudo tee -a /var/www/web/sites/default/settings.local.php
+          sudo chmod 777 /var/www/web/db.sqlite
```

Comment:
I don't think we need to configure the DB settings for SQLite -- this happens as part of the `post-create-project-cmd` script defined in `composer.json`.